src/shogun/distance/EuclideanDistance.h

```diff
@@ -37,7 +37,12 @@ namespace shogun
  * \f[\displaystyle
  *  d({\bf x},{\bf x'})= \sum_{i=0}^{n}|{\bf x_i}-{\bf x'_i}|^2
  * \f]
- *
+ * 
+ * Alternatively can be computed as :  
```

Comment:
I wasnt completely sure of implications on other algorithms so decided to start off by keeping it optional.